lessons/32-js-destructure/index.js

```diff
@@ -0,0 +1,13 @@
+const favFruits = prompt("Please enter you top 5 favorite fruits (in order), separated by a comma: ");
+
+const arrFruits = favFruits.split(', ');
+
+console.log(arrFruits);
+
+const [firstFruit, secondFruit, thirdFruit] = arrFruits;
+const [, , , ...restOfFruit] = arrFruits;
```

Comment:
Only suggestion is that you can do the spread inside of the first line here to keep it short but this works too!

`const [firstFruit, secondFruit, thirdFruit, ...restOfFruit] = arrFruits;`